getting-started/FAQ.md

```diff
@@ -289,6 +289,8 @@ EOF
 docker exec -it kind-control-plane bash -c "apt update && apt install apparmor-utils -y && systemctl restart containerd"
```

Comment:
Please update this FAQ to generic way as well. There are cases where AppArmor is availble but not enabled. We need to instruct them to install apparmor utils.

Example CBL Mariner. You can see the updater script we install apparmor utils there as well.

So the goal is to update the FAQ in a generic manner such that people can understand why do they need to enable AppArmor utils, how to check whether apparmor availability.

Why is the additional logic is needed for dockerized nodes.